pypdf/generic/_appearance_stream.py

```diff
@@ -329,13 +363,10 @@ def __init__(
             text,
             selection,
             rectangle,
-            font_descriptor,
-            font_glyph_byte_map,
-            font_name,
-            font_size,
-            font_color,
-            is_multiline,
-            alignment
+            font_descriptor=font_descriptor,
+            font_glyph_byte_map=font_glyph_byte_map,
+            font_name=font_name,
+            **formatting_kwargs
```

Comment:
Sorry, this is not what I meant. I was referring to passing all arguments as keyword arguments by its name instead (like you did for the first ones).